companies/mi-com.json

```diff
@@ -0,0 +1,32 @@
+{
+    "slug": "mi-com",
+    "relevant-countries": [
+        "all"
+    ],
+    "name": "Xiaomi Communications Co., Ltd.",
+    "runs": [
+        "Xiaomi Technology Netherlands B.V.",
+        "Xiaomi Singapore Pte. Ltd."
+    ],
+    "address": "c/o Wework Room 04-106\nStrawinskylaan 4117 4th Floor\n1017 ZX Amsterdam\nNetherlands",
+    "email": "privacy@xiaomi.com",
```

Comment:
Any reason not to suggest email as the transport medium? :)

Reply:
good catch! Added.